### ISSUE TITLE
ODataClient generate incorrect "Accept" header

### ISSUE THREAD

**Issue by kingcomxu** [object/Simple.OData.Client#129]
```
public static ODataClient GetODataClient()
        {
            try
            {
#if DEBUG
                var odataClient = new ODataClient(new ODataClientSettings("http://localhost.fiddler:8888/odata/")
#else
                var odataClient = new ODataClient(new ODataClientSettings(fx.Config.ServiceRoot + "odata/")
#endif
                {
                    BeforeRequest = (req) =>
                    {
                        req.Headers.Add("FXSession", fx.LoginContext.SessionID);
                    },

                    PayloadFormat = ODataPayloadFormat.Json,
                    ReuseHttpConnection = true,
                    //MetadataDocument = ODataClientMetadataDocumentString,
                    // OnTrace = (x, y) => { Debug.WriteLine(string.Format(x, y)); },
                });

                //if (string.IsNullOrEmpty(ODataClientMetadataDocumentString))
                //{
                //    try
                //    {
                //        ODataClientMetadataDocumentString =await odataClient.GetMetadataDocumentAsync();
                //    }
                //    catch (Exception ex)
                //    {
                //        throw ex;
                //    }
                //}

                return odataClient;
            }
            catch (Exception ex)
            {
                throw ex;
            }
        }
```

Useage:
```

var client = fx.GetODataClient();
            PropertyDetail ret = new PropertyDetail(false);
            ret.TaskProperty = client.For<FKProperty>().Key(propertyId).FindEntryAsync();
/*header:
...
Accept: application/json, application/xml, application/text
...
*/
            ret.TaskPropertyExt = client.For<FKPropertyExt>().Filter(row => row.PropertyId == propertyId).FindEntryAsync();
/*header:
...
Accept: application/json, application/xml, application/text, application/json, application/xml, application/text
...
*/
            ret.TaskPhotos = client.For<FKPhoto>().Filter(row => row.PropertyId == propertyId).FindEntriesAsync();
            ret.TaskKeyHistorys = client.For<FKPropertyKeyHistory>().Filter(row => row.PropertyId == propertyId).FindEntriesAsync();
            ret.TaskRecommandHistorys = client.For<FKPropertyRecommandHistory>().Filter(row => row.PropertyId == propertyId).FindEntriesAsync();
            ret.TaskFollows = client.For<FKFollow>().Filter(row => row.PropertyId == propertyId).FindEntriesAsync();
            ret.TaskOperationLogs = client.For<OperationLog>().Filter(row => row.EntityId == propertyId).FindEntriesAsync();
/*
...
Accept: application/json, application/xml, application/text, application/json, application/xml, application/text, application/json, application/xml, application/text, application/json, application/xml, application/text, application/json, application/xml, application/text, application/json, application/xml, application/text, application/json, application/xml, application/text
...
*/

            return ret;
```

**Comment by object** [comment]
Thank you for the issue report. I will look into it soon.

**Comment by kingcomxu** [comment]
my application use cefsharp.wpf.ChromiumWebBrowser and I'm invoke GetODataClient in webbrowser's javascript, it look like a multithreading  problem
I can't reproduce this bug in console application, even I invoke GetODataClient() in a new Thread
sorry for my bad english

**Comment by object** [comment]
I have reproduced the error. Fix will come soon.

**Comment by object** [comment]
Fixed in 4.9.3.
